README.md

```diff
@@ -1 +1,39 @@
-# hello-microverse-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+First Microverse project to master all of the tools and best practices previously learned.
+
+
+## Built With
+
+- HTML/CSS
+- Git commands
+
+
+## Authors
+
+👤 **Author1**
+
+
+- GitHub: [@lucascomamala](https://github.com/lucascomamala)
+- Twitter: [@ComamalaLucas](https://twitter.com/ComamalaLucas)
+- LinkedIn: [Lucas Comamala](https://linkedin.com/in/lucas-comamala)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] You have not added the MIT.md file in your directory. This file is very important because it is a license file for your project. Don't worry, follow these steps to create it :receipt: :+1:

- First, create MIT.md file in your main directory.
- Then put contents into your MIT.md from this [file](https://github.com/HAMMAS-SALEEM/Hello-Microverse/blob/master/MIT.md).
- At last, add your name, and year.

**Now, you have successfully created a license file.** :tada: